nominal/cli/auth.py

```diff
@@ -11,6 +14,22 @@ def auth_cmd() -> None:
     pass
 
 
+def _validate_token_url(token: str, base_url: str) -> None:
+    """Ensure the user sets a valid configuration before letting them import the client."""
+    token_link = "https://app.gov.nominal.io/settings/user?tab=tokens"
+    r = requests.get(f"{base_url}/openapi", headers={"Authorization": f"Bearer {token}"})
```

Comment:
We have `NominalClient.get_user()` which is being used in some scripts to do this eager check--let's use that instead of `/openapi` which I don't think is particularly concrete comparatively

Reply:
nice thanks. i couldn't find that but it's much better

Reply:
@alkasm we can't call `get_user` here because the url and token haven't been set yet. That assumes a correctly setup auth, which requires this flow to complete.

Reply:
`NominalClient.create(base_url, token).get_user()`?

Reply:
thanks, way cleaner